manifests/app.pp

```diff
@@ -44,37 +43,37 @@
   case $install_provider {
     'http': {
       fail('Moodle http install not supported at this time.')
-    # # manage the staging class
-    # class { 'staging':
-    #   path  => '/var/staging',
-    # }
-    #
-    # # download the staged file
-    # staging::file { 'moodle.tgz':
-    #   source => $download_url,
-    # }
-    #
-    # # ensure that the directory
-    # if !defined(File[$install_dir]) {
-    #   file { $install_dir:
-    #     ensure => directory,
-    #     owner  => $www_owner,
-    #     group  => $www_group,
-    #     mode   => '0755',
-    #   }
-    # }
-    #
-    # # get the parent directory... the moodle distribution
-    # # gets extracted to a 'moodle' directory
-    # $install_parent = getparent($install_dir)
-    #
-    # staging::extract { 'moodle.tgz':
-    #   target  => $install_parent,
-    #   user    => $www_owner,
-    #   group   => $www_group,
-    #   creates => "${install_dir}/install.php",
-    #   require => [Staging::File['moodle.tgz'],File[$install_dir, $dataroot]],
-    # }
+# # manage the staging class
```

Comment:
Perhaps don't change the indentation on this comment? Ideally, it should be removed entirely (maybe kept in a separate branch,) but changing the indentation will make it harder to figure out where this commented out code is supposed to slot in